src/applications/personalization/rated-disabilities/actions/index.js

```diff
@@ -36,12 +53,24 @@ export function fetchTotalDisabilityRating() {
     );
 
     if (response.errors) {
-      // record event here
+      const errorCode = response.errors[0].code;
+      if (serverErrorRegex.test(errorCode)) {
+        recordEvent({
+          event: `${DISABILITY_PREFIX}-combined-load-failed`,
+          'error-key': '500 internal error',
+        });
+      } else if (serviceErrorRegex.test(errorCode)) {
+        recordEvent({
+          event: `${DISABILITY_PREFIX}-combined-load-failed`,
+          'error-key': '401 no combined rating found',
```

Comment:
this is fine as long as you're fine with other 400 errors being reported as 401 errors.

Reply:
This one could _potentially_ change, since the vets-api endpoint for this hasn't been constructed yet. I think we decided to go with the current value for the time being, since the plan was to try and keep it consistent with what we also get back for the list.